### PR TITLE
Updates to Red Beginner Routes & FAQ

### PR DESCRIPTION
**Pull request by SquareSR**
FAQ now includes a link to the Red/Blue Strategy Wiki
Lass/nolass routes have some helpful embedded images and more links to images.
Classic beginner now has links to some gym leader locations and more indicators for where to fly such as (toggle down X) 
Also improved lass/nolass description of Boom Strats to avoid ppl thinking they should use a revive after Weezing. 
Added a note that items should be bought in the order that they are listed in the guide.

### CODE REVIEW

**Comment by SquareSR** [comment]
I also moved EQ Menu to inside Safari Zone, since this was found to be optimal and gives more wiggle room on repel steps for beginners. I did NOT move EQ menu for red classic since we use rare candies in EQ menu and it's going to be tricky not having cursor on Nidoking.

**Review by TiKevin83 — APPROVED** [review]

**Comment by SquareSR** [comment]
Overview: Strategy Wiki linked in FAQ, Images added to Red Glitchless Beginner Guides, Note added to both Classic and Glitchless beginner guides about buying items in the correct order, EQ Menu moved to Safari Zone for Glitchless guides, and lastly embedded Boom Strats into the Intermediate route just as other HP Strats already are.

File: docs/gen-1/red-blue/main-glitchless/WR-Route/README.md [hunk 1]
```diff
@@ -352,15 +352,21 @@ Cycling Road
 - Get the PPUP
 
 Post Cycling Road Menu:
-- Use Super Repel (skip repeling if we only bought 9)
+- Bike
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/WR-Route/README.md]: That seems a bit light for a "menu", maybe reword this

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md [hunk 2]
```diff
@@ -1,15 +1,25 @@
 # Pokemon Red Glitchless Beginner Guide, Lass Version
 
+# Pokemon Red Glitchless Beginner Guide, No-Lass Version
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: to remove

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md [hunk 3]
```diff
@@ -91,12 +102,14 @@ Options: Battle Style to Set before leaving Brock's gym
 - Caterpie: Leer + HA x2 (+ Tackle)
 - Weedle: Leer + HA x2
 	- Leer + HA + Tackle if you got String Shot hit OR Tackle will always kill
-- Caterpie: HA x2 + Tackle
+- Caterpie: Leer + HA + Tackle
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: This was HA x2 because pp is supposedly not a problem on lass

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md [hunk 4]
```diff
@@ -450,27 +480,31 @@ Get the Poke Flute and Fly to Celadon City
 
 ## Celadon City
 
-- Use the center
-- Bike to Saffron City
+- MUST Take the center in Celadon. Replenishes PP & sets Celadon as the warp point.
+- Bike East to Saffron City, trading Fresh Water for passage, and enter Silph Co.
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: "enter Silph Co" is said twice :nerd:

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md [hunk 5]
```diff
@@ -707,9 +767,9 @@ After Lance:
 - [OPTIONAL SAVE: Saving for Champ can be good if you don't have a spare X Accuracy + X Special + Revive]
 
 **Champion:**
-- Pidgeot: X Spec
-	- If Pidgeot does NOT use Sky Attack turn 1:
-		- Pidgeot: X Acc + HD
+Pidgeot: **Turn 1: USE X SPECIAL**, then adapt:
+	- If Pidgeot does NOT use Sky Attack on turn 1:
+		- Pidgeot: use X Acc on turn 2, HD
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: You broke the formatting, on top of that i'm not sure adding that many words make it clearer

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md [hunk 6]
```diff
@@ -1,15 +1,25 @@
...
 	- [Optimal](../../resources/nido-manip.md) (HIGHLY RECOMMENDED)
-	- [1 A Press](https://youtu.be/2bwQZ6jPxRE)
+	- [Easy - 1 A Press](https://youtu.be/2bwQZ6jPxRE)
 - Mount Moon Manip:
-	- [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (recommended for first runs)
-		- [Older version](https://pastebin.com/7kcZu3g4) (easier path, but select yoloball)
+	- [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (recommended)
+		- [Easy - Older version](https://pastebin.com/7kcZu3g4) (easier path, but select yoloball)
 	- [Post Nerd Backup Paras](../../resources/postnerd-backup-paras.md)
+	- [Route 3 Moon Manip](https://pastebin.com/tggXpQRC)
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: This was not listed on the lass document because its simply not usable with lass, i'm fine letting this here but hopefully it doesnt cause any confusion

File: docs/gen-1/red-blue/main-glitchless/beginner-route/nolass/README.md [hunk 7]
```diff
@@ -129,7 +139,7 @@ Catch a flyer:
...
+- Do not learn a full Moon Manip for your first runs, start by learning [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (or the easier [older version](https://pastebin.com/7kcZu3g4)) and walk through the first floor normally ([movement map](https://i.imgur.com/xzgmNBk.jpeg)). You will choose later if you want to learn [Route 3 Moon](https://pastebin.com/tggXpQRC) (optimal for this route) or [Entr Moon](https://pastebin.com/jnj9j47S) (slightly less optimal, but still perfectly fine). There is also a very easy [Post Nerd Backup Paras](../../resources/postnerd-backup-paras.md) you can learn to manip a Paras just before exiting Mt. Moon.
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/nolass/README.md:142]: Why is this comment different on both files, should be copied

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md [hunk 8]
```diff
@@ -630,16 +686,20 @@ Menu after leaving the gym:
...
 
-[Super Repel locations](https://imgur.com/gallery/laeW1PT) and [Victory Road map](https://imgur.com/gallery/vKQYRQZ)
+- After the first badge check, get on the bike.
+
+[Next two Super Repel locations](https://imgur.com/gallery/laeW1PT) and [Victory Road map](https://imgur.com/gallery/vKQYRQZ)
 
 - Pick up the [hidden Max Ether](https://gunnermaniac.com/pokeworld?map=34#8/90) from the water
-- Use a Super Repel one step into the cave
-- Watch a top leaderboard run for how to do the Strength boulder puzzles
+- Exit water, super repel, and get on the bike.
```
Comment (Grogir) [inline docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: Detail but would appreciate upper case Super Repel

**Comment by Grogir** [comment]
Personally not a fan of adding so many boom strats tables in the intermediate guide, but im not going to force my view

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md (hunk 3 above)
Comment (SquareSR) [reply docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: This one I am still in favor of changing since I've seen some beginners go from Beginner Lass to No-Lass, but continue to HAx2 the Caterpie not realizing they are wasting an HA. Perhaps giving the option to do either is good I think

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md (hunk 5 above)
Comment (SquareSR) [reply docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: Got it, reverting this back entirely to how it was.

File: docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md (hunk 6 above)
Comment (SquareSR) [reply docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md]: Oh that makes sense, I agree. Copy-paste error here on my part, will remove it.

File: docs/gen-1/red-blue/main-glitchless/beginner-route/nolass/README.md (hunk 7 above)
Comment (SquareSR) [reply docs/gen-1/red-blue/main-glitchless/beginner-route/nolass/README.md:142]: Understood - will make wording match except for the disclaimers regarding Rt.3 and Entr needing to differ.
